common/src/main/java/com/wynntils/features/ui/CustomSeaskipperScreenFeature.java

```diff
@@ -21,6 +22,9 @@
 @ConfigCategory(Category.UI)
 public class CustomSeaskipperScreenFeature extends Feature {
 
+    @RegisterConfig("feature.wynntils.customseaskipperscreen.screentype")
```

Comment:
Why the custom key? If you rename the variable to `screenType` the name won't conflict (if that was the problem originally).

Reply:
Probably not needed, I think wherever I copied it from originally had it so thought it need it